src/dynamics/video_player/player/player.js

```diff
@@ -1635,6 +1635,21 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
 
                 },
 
+                _updatePlaylistByParams: function() {
+                    const playlist = [...this.get("playlist")];
+                    const isVideoAlreadyInPlaylist = playlist
+                        .some((v) => v.title === this.get("title") && v.source === this.get("source"));
```

Comment:
Hi @Josewd, title isn't mandatory, so this won't work if we only have source. A video can also have multiple sources, if the one being played isn't the "main" one it's possible this will cause the video to show up twice on sidebar. 😬